deploy.py

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env python3
-# Copyright 2019 Google LLC
+#!/usr/bin/env py_virtual_env/bin/python3
```

Comment:
No need for `env` now that we know the path.
```suggestion
#!py_virtual_env/bin/python3
```

Reply:
Done